src/NUnitFramework/nunitlite-runner/nunitlite-runner-4.5.csproj

```diff
@@ -25,7 +25,8 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Prefer32Bit>false</Prefer32Bit>
-    <Externalconsole>true</Externalconsole>
+    <Commandlineparameters>nunit.framework.tests.dll</Commandlineparameters>
+    <ConsolePause>false</ConsolePause>
```

Comment:
MonoDevelop stores the Startup program in the project file, while VS doesn't. It does no harm.